### ISSUE TITLE
Windows - provider called even if mouse does not move

### ISSUE THREAD

**Issue by lloiser** [facebooknuclide/hyperclick#30]
### Issue and Steps to Reproduce

On windows hyperclick providers are called continuously even though I do not move the mouse cursor at all.
This does not happen on macOS. 
Here is a gif that demonstrates that pressing the trigger keys and not moving the mouse still calls the [flow-ide](https://github.com/steelbrain/flow-ide) hyperclick provider

Demo:
![win](https://user-images.githubusercontent.com/3003233/28864067-9657aa14-776b-11e7-9031-73b233c56ad0.gif)

Main issue: https://github.com/steelbrain/flow-ide/issues/103

The hyperclick provider of flow-ide: https://github.com/steelbrain/flow-ide/blob/v1.7.0/lib/index.js#L253-L314

### Expected Behavior

Only called once because my mouse has not moved.

### Actual Behavior

The hyperclick provider is flooded with calls. 

### Versions
* Atom: 1.18.0
* OS: Windows 10

### Additional Details
* Installed packages (`apm ls --installed`):
```
Community Packages (6) C:\Users\xyz\.atom\packages
├── busy-signal@1.4.3
├── flow-ide@1.7.0
├── hyperclick@0.1.3
├── intentions@1.1.5
├── linter@2.2.0
└── linter-ui-default@1.6.3
```

**Comment by hansonw** [comment]
aha - looks like Windows' key repeat feature works for modifier keys too :) Should be pretty easy to fix this case.

**Comment by hansonw** [comment]
Should be fixed in v0.1.5.
